nc/data/download_from_nc.py

```diff
@@ -13,6 +14,12 @@
 def show_ftp_listing(s):
     logger.debug(s)
 
+def ftps_connect(host):
+    ftps = FTP_TLS() 
+    ftps.ssl_version = ssl.PROTOCOL_SSLv23 
+    ftps.connect(host) 
+    ftps.login(os.environ.get('NC_FTP_USER'), os.environ.get('NC_FTP_PASSWORD')) 
```

Comment:
I know it used to be this way, but I'd prefer if we moved all 3 ``NC_*`` variables to settings to match our general workflow for handling env variables.